<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>목표를 입력하세요.</string>
  </property>
  <widget class="QLineEdit" name="txt_goal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tmon몬소리 Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>목표량을 입력해 주세요.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancle">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>98</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="main_frame.qrc">:/email/img/img_email.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>txt_goal</zorder>
  <zorder>label</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_cancle</zorder>
 </widget>
 <resources>
  <include location="main_frame.qrc"/>
 </resources>
 <connections/>
</ui>
